css/templates/e-commerce/elements.css

```diff
@@ -0,0 +1,15 @@
+@import '../../core';
+@import 'page-vars';
```

Comment:
Can we look to get a solution soon? Just with this one page the CSS is over 200 lines of code.